SoftLayer/CLI/block/refresh.py

```diff
@@ -9,10 +9,15 @@
 @click.command(cls=SoftLayer.CLI.command.SLCommand, )
 @click.argument('volume_id')
 @click.argument('snapshot_id')
+@click.option('--force-refresh', '-f',
+    help = "Cancel current refresh process and initiates the new refresh.",
+    type = click.BOOL,
```

Comment:
```suggestion
    is_flag=True,
```

Have this option set to a flag which will allow users to do `slcli block refresh VOLUMEID SNAPSHOTID --force-refresh` instead of `slcli block refresh VOLUMEID SNAPSHOTID --force-refresh 1`

Also, these lines need to be aligned with the first `'` on line 12, which is what tox analysis is complaining about

Reply:
@allmightyspiff i agree this is more simpler, but our API's dev want existing way only, Is it fine we do this way passing value along with param?

Reply:
When setting `is_flag=True` `force-refresh` will be a boolean value, and it will make the same api command. This simply makes the command easier for the users.   If `--force-refresh` is present as an option, it will be True, if it is not present, it will be False.